test/regression.uts

```diff
@@ -13041,6 +13041,62 @@ p[IP]
 ############
 + PacketList methods
 
+= sr()
+
+class Req(Packet):
```

Comment:
Could you use an existing Packet instead? For example, `IP()/ICMP()ˋ?
I am ok for using these specific ones. I am just curious.